middleware/package.json

```diff
@@ -27,13 +27,14 @@
   "author": "TODO",
   "dependencies": {
     "@projectriff/message": "^1.0.0",
-    "@salesforce/salesforce-sdk": "^1.3.0"
+    "@salesforce/salesforce-sdk": "^1.3.0",
+    "tslib": "^1.11.2"
```

Comment:
What made us move `tslib` from `devDeps` to `deps`?

Reply:
Shouldn't tslib always be a dependency? It is a runtime library.

Reply:
I'm not sure. I guess maybe it worked before because of other things bringing it in? I'm 👍 , just like to call out dep location changes in case they were accidental.